html/page2.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="/css/page2.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Amatic+SC:wght@700&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="form-popup" id="myForm">
+      <form action="" class="form-container">
+        <h1 class="heading">Creat new task</h1>
+        <hr color="white" width="100%" />
+
+        <div class="formInputs">
+          <label for="tittle" class="label">Tittle</label>
+          <input
+            type="text"
+            name="tittle"
+            id="tittleInput"
+            placeholder="Enter the title of the task"
+            required
+          />
+
+          <label for="date" class="label">Date</label>
+          <input type="date" name="date" id="dateInput" required />
+
+          <label for="time" class="label">Time</label>
+          <input type="time" name="time" id="timeInput" required />
+
+          <label for="description" class="label">Description</label>
+          <input
+            type="text"
+            name="description"
+            id="descriptionInput"
+            placeholder="Enter the description of the task"
+            required
+          />
+
+          <label for="notes" class="label">Notes</label>
+          <input
+            type="text"
+            name="notes"
+            id="notesInput"
+            placeholder="Enter additional notes"
+          />
+        </div>
+
+        <div class="button">
+          <a href="" id="cancel">Cancel</a>
+          <button type="submit" id="submitButton">Done</button>
```

Comment:
why you add the cancel button as a hyperlink instead of a button

Reply:
I didn't found a button type attribute (cancel) and when I research about it I found that it's recommend to make it as hyperlink and style it to look like button.

Reply:
I didn't found a button type attribute (cancel) and when I research about it I found that it's recommended to make it use hyperlink and style it to look like a button.
Should I make it like ````<button><a href="main.html">cancel</a></button>```` ?
Or what's your recommendation?

Reply:
you can add it like  `<input type="button" name="cancel" value="cancel" onClick="window.location.href='main.html" />`

Reply:
Ok, thanks